loginhooks.go

```diff
@@ -0,0 +1,59 @@
+package maildoor
+
+import (
+	"net/http"
+	"time"
+)
+
+const (
+	// DefaultCookieName is the of the cookie that
+	// Maildoor will use to store the user's email address.
+	// when the default login/logout functions are not overridden.
+	DefaultCookieName     = "maildoor-cookie"
+	defaultCookieDuration = 7 * 24 * time.Hour
```

Comment:
@paganotoni Why the duration is private? Don't you think making it editable could be beneficial in case others won't agree about the 7 days duration?

Reply:
I think if we want to make this configurable we should use the Options pattern.

Reply:
Okay, that makes sense to me...